addons/mail/static/src/core/common/attachment_list.xml

```diff
@@ -13,81 +21,91 @@
             <div class="d-flex flex-grow-1 flex-wrap mx-1 align-items-center" t-att-class="{
                 'justify-content-end': isInChatWindowAndIsAlignedRight and !env.inComposer,
             }" role="menu" >
-                <t t-foreach="this.props.attachments" t-as="attachment" t-key="attachment.id">
-                    <div t-if="attachment.isImage"
-                        t-att-aria-label="attachment.filename"
-                        class="o-mail-AttachmentImage d-flex position-relative flex-shrink-0 mw-100 mb-1 me-1"
-                        t-att-title="attachment.name"
-                        t-att-class="{ 'o-isUploading': attachment.uploading }"
-                        tabindex="0"
-                        t-att-data-mimetype="attachment.mimetype"
-                        t-on-click="() => this.fileViewer.open(attachment, props.attachments)"
-                        role="menuitem"
-                    >
-                        <img
-                            class="img img-fluid my-0 mx-auto o-viewable rounded"
-                            t-att-class="{ 'opacity-25': attachment.uploading }"
-                            t-att-src="getImageUrl(attachment)"
-                            t-att-alt="attachment.name"
-                            t-attf-style="max-width: min(100%, {{ imagesWidth }}px); max-height: {{ props.imagesHeight }}px;"
-                            t-on-load="onImageLoaded"
+                <t t-foreach="props.attachments" t-as="attachment" t-key="attachment.id">
+                    <div
+                        t-on-click.capture="(ev) => this.select(attachment, ev)"
+                        t-att-class="{ 'position-relative border p-2 pt-4': props.selectionMode, 'border-primary': selected(attachment) }">
+                        <CheckBox
+                            t-if="props.selectionMode"
+                            className="'position-absolute top-0 end-0'"
+                            onChange="() => this.select(attachment)"
```

Comment:
Missing `ev` param, which causes `Cannot stopPropagation of undefined` error on clicking on a single checkbox

Reply:
onChange is a props, it doesn't have an `ev`. 
I can just guard `ev`